massa-pool-worker/src/tests/mod.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2022 MASSA LABS <info@massa.net>
 
-mod config;
-mod mock_protocol_controller;
-mod operation_pool_tests;
-mod scenario;
-pub(crate) mod tools;
+//mod config;
```

Comment:
I don't get this.

Reply:
can you recheck this with the latest conflict resolution ?